sdk/src/test/java/io/opentelemetry/sdk/metrics/view/SummaryAggregationTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.metrics.view;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import io.opentelemetry.sdk.metrics.aggregator.DoubleMinMaxSumCount;
+import io.opentelemetry.sdk.metrics.aggregator.LongMinMaxSumCount;
+import io.opentelemetry.sdk.metrics.common.InstrumentType;
+import io.opentelemetry.sdk.metrics.common.InstrumentValueType;
+import io.opentelemetry.sdk.metrics.data.MetricData.Descriptor.Type;
+import org.junit.Test;
+
+public class SummaryAggregationTest {
```

Comment:
The aggregation should be also renamed to MinMaxSumCount :)

Reply:
Will change

Reply:
done